### ISSUE TITLE
How could I set AUR mirror for yay

### ISSUE THREAD

**Issue by nouh** [Jguer/yay#623]
#### Steps to reproduce

1.
2.
3.

**Comment by parkerlreed** [comment]
What? Yay is only designed to pull from official AUR. What do you mean by AUR mirror?

It'd help if you actually filled out the template with details.

**Comment by Morganamilo** [comment]
They probably mean running a local aurweb instance https://git.archlinux.org/aurweb.git/. Currently the only way is to edit the source and send us a patch ;)

**Comment by codehz** [comment]
yaourt support this feature.
 /etc/yaourtrc AURURL option

**Comment by wangqr** [comment]
In some locations it is slow or not possible to connect to the official AUR. Thats why we use a mirror. For example we have [TUNA mirror](https://aur.tuna.tsinghua.edu.cn/) in China.

**Comment by amaothree** [comment]
Yeah, this function is necessary for us, because the f**king GFW. :)

**Comment by Morganamilo** [comment]
Now I understand, The original issue should have said this in the first place. Added to my todo then.
